bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/relay.xml

```diff
@@ -431,91 +453,77 @@
 		<item-type>Number:Power</item-type>
 		<label>Watt</label>
 		<description>Current power consumption in Watt</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.2f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="meterAccuWatts" advanced="true">
 		<item-type>Number:Power</item-type>
 		<label>Accumulated Watt</label>
 		<description>Accumulated current power consumption in Watt from all meters</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.2f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="meterAccuTotal" advanced="true">
 		<item-type>Number:Power</item-type>
 		<label>Accumulated Total</label>
 		<description>Accumulated total power consumption in kw/h from all meters</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.3f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="meterAccuReturned" advanced="true">
 		<item-type>Number:Power</item-type>
 		<label>Accumulated Returned</label>
 		<description>Accumulated returned power consumption in kw/h from all meters</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.3f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="meterReactive">
 		<item-type>Number:Power</item-type>
 		<label>Reactive Watt</label>
 		<description>Instantaneous reactive power in Watts (W)</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.3f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="lastPower1" advanced="true">
 		<item-type>Number:Energy</item-type>
 		<label>Last Power #1</label>
 		<description>Last power consumption #1 - one rounded minute</description>
-		<state readOnly="true" pattern="%f %unit%">
-		</state>
-	</channel-type>
-	<channel-type id="lastPower2" advanced="true">
-		<item-type>Number:Energy</item-type>
-		<label>Last Power #2</label>
-		<description>Last power consumption #2 - one rounded minute</description>
-		<state readOnly="true" pattern="%f %unit%">
-		</state>
-	</channel-type>
-	<channel-type id="lastPower3" advanced="true">
-		<item-type>Number:Energy</item-type>
-		<label>Last Power #3</label>
-		<description>Last power consumption #3 - one rounded minute</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.3f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="meterTotal">
 		<item-type>Number:Energy</item-type>
 		<label>Total Energy</label>
 		<description>Total power consumption in kw/h</description>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%.3f %unit%">
 		</state>
 	</channel-type>
 	<channel-type id="meterReturned">
 		<item-type>Number:Energy</item-type>
 		<label>Total Returned Energy (kw/h)</label>
```

Comment:
The unit shouldn't be mentioned as the user can also decide to display the value in Wh.
```suggestion
		<label>Total Returned Energy</label>
```

Reply:
removed all kw/h